src/v3/src/components/ImageWithText/ImageWithText.tsx

```diff
@@ -14,12 +14,15 @@ import './style.module.css';
 
 import { Box, Typography } from '@okta/odyssey-react-mui';
 import { h } from 'preact';
-import { ImageWithTextElement, UISchemaElementComponent } from 'src/types';
+import { ImageName, ImageWithTextElement, UISchemaElementComponent } from 'src/types';
+
+import { getImageByName } from '../../util';
 
 const ImageWithText: UISchemaElementComponent<{
   uischema: ImageWithTextElement
 }> = ({ uischema }) => {
-  const Icon = uischema.options.SVGIcon;
+  const { options: { SVGIcon } } = uischema;
+  const Icon = typeof SVGIcon === 'function' ? SVGIcon : getImageByName(SVGIcon as ImageName);
```

Comment:
Any reason this cant just stay a FunctionComponent type and in the transformers you can directly call the `getImageByName` util function passing the result to the SVGIcon?

Reply:
absolutely right dont even need the util function its unnecessary, we can just import the the Image components directly in the transformers and pass them into the options.